erroranalysis/tests/test_cohort_filter.py

```diff
@@ -226,7 +229,38 @@ def test_cohort_filter_index(self):
         model_task = ModelTask.CLASSIFICATION
         model = create_sklearn_svm_classifier(X_train, y_train)
         categorical_features = []
-        model_task = ModelTask.CLASSIFICATION
+        run_error_analyzer(validation_data,
+                           model,
+                           X_test,
+                           y_test,
+                           feature_names,
+                           categorical_features,
+                           model_task,
+                           filters=filters)
+
+    def test_cohort_filter_regression_error(self):
+        X_train, X_test, y_train, y_test, feature_names = \
+            create_diabetes_data()
+        X_train = pd.DataFrame(X_train, columns=feature_names)
+        X_test = pd.DataFrame(X_test, columns=feature_names)
+
+        # filter on regression error, which can be done from the
+        # RAI dashboard
+        filters = [{'arg': [40],
```

Comment:
nit: use arg, column, method from constants in erroranalysis package:

https://github.com/microsoft/responsible-ai-toolbox/blob/main/erroranalysis/erroranalysis/_internal/constants.py#L6

Reply:
I have a separate work item for this. Will address this in a future PR.